packages/itwinui-react/vitest.config.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { defineConfig } from 'vitest/config';
+import * as path from 'node:path';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: 'setupTests.ts',
+    exclude: ['/node_modules/', '/esm/', '/cjs/'],
+    reporters: ['junit', 'default'],
+    outputFile: 'coverage/junit.xml',
+    alias: [
+      {
+        find: /^(.*)\/styles.js$/,
+        replacement: path.resolve('./__mocks__/styles.js'),
+      },
+    ],
+    typecheck: {
+      enabled: true,
+      include: ['**/*.test.?(c|m)[jt]s?(x)'],
```

Comment:
I think this actually broke our tests. The tests are no longer being run; they are only being type-checked.